dotfiles_template/README.md

```diff
@@ -4,10 +4,22 @@
 
 ## Restore your Dotfiles
 
-* Install git
-* Clone your dotfiles repository `git clone [your repository of dotfiles] $HOME/.dotfiles`
-* Go to your dotfiles folder `cd $HOME/.dotfiles`
-* Install git submodules `git submodule update --init --recursive`
-* Install your dotfiles `DOTFILES_PATH="$HOME/.dotfiles" DOTLY_PATH="$DOTFILES_PATH/modules/dotly" "$DOTLY_PATH/bin/dot" self install`
-* Restart your terminal
-* Import your packages `dot package import`
+```bash
+export GITHUB_USER=""
+export DOTFILES_REPOSITORY="dotfiles"
+export DOTFILES_PATH="${HOME}/.dotfiles"
+export DOTLY_PATH="$DOTFILES_PATH/modules/dotly"
+#ssh-keygen # Uncoment to generate your ssh key
+#cat ~/.ssh/id_rsa.pub | pbcopy # Uncoment if you have generated a new key
+#      In Linux use 'xclip -selection clipboard'
+# Add to Github > Settings > SSH and GPG Keys > Add new SSH key
+xcode-select --install
+#rm -rf "$DOTFILES_PATH" "~/.dotly" # Optional step
+git clone git@github.com:$GITHUB_USER/$DOTFILES_REPOSITORY.git "$DOTFILES_PATH"
+cd "$DOTFILES_PATH"
+git submodule update --init --recursive
+"$DOTLY_PATH/bin/dot" self install
+"$DOTLY_PATH/bin/dot" package import # Import your packages
+echo "Press a key to exit the terminal and later reopen it..."
+exit
+```
```

Comment:
We could add all of this commands to a `script` and then execute it 😊

Reply:
Yes, its a work in progress in my repo but I'm out of time until half february. I was thinking about a remote "restorer" like the installer that do all the stuff.

Reply:
There is also a mac command to add if you approve PR #67

```bash
"$DOTLY_PATH/bin/dot" mac defaults import
```